pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -421,11 +421,17 @@ func TestPriorityQueue_AssignedPodAdded(t *testing.T) {
 		Spec: v1.PodSpec{NodeName: "machine1"},
 	}
 
-	q := createAndRunPriorityQueue(newDefaultFramework())
+	timestamp := time.Now()
+	c := clock.NewFakeClock(timestamp)
+	q := createAndRunPriorityQueue(newDefaultFramework(), WithClock(c))
 	q.Add(&medPriorityPod)
 	// Add a couple of pods to the unschedulableQ.
 	addOrUpdateUnschedulablePod(q, q.newPodInfo(&unschedulablePod))
 	addOrUpdateUnschedulablePod(q, q.newPodInfo(affinityPod))
+
+	// Override clock to exceed the DefaultPodInitialBackoffDuration so that unschedulable pods
+	// will be moved to activeQ
+	c.SetTime(timestamp.Add(DefaultPodInitialBackoffDuration + 1))
```

Comment:
Use `c.Step` instead.

Reply:
But we shouldn't be doing this. Is there a clearBackoff related to this that we removed? If so, we should probably also reset Attempts here? @Huang-Wei